src/utils/Utils.js

```diff
@@ -25,6 +25,7 @@ export default class Utils {
   static loadUnusedPairs(allProps, properties) {
     const resultMap = Object.assign({}, allProps);
     properties.forEach((prop) => {
+      if (resultMap[prop.type.namespace] === undefined) return;
```

Comment:
This will let `null` and empty namespaces through. It's better to do `if (!resultMap[prop.type.namespace])`. That will pick up `null`, `undefined` (those are different in JS), `false`, and empty strings.